templates/dependapanda.text.erb

```diff
@@ -1,4 +1,4 @@
-<%= @all_prs_count > 1 ? "You have #{@all_prs_count} Dependabot PRs open on the following apps:" : "You have 1 Dependabot PR open on the following app:" %>
+<%= @all_prs_count > 1 ? "You have #{@all_prs_count} Dependabot/Renovate PRs open on the following apps:" : "You have 1 Dependabot/Renovate PR open on the following app:" %>
```

Comment:
Could we change the wording to make it tool agnostic? We currently don't have any docs on Renovate so we don't want to surprise/confuse the teams. The test will also need updating accordingly.

```suggestion
<%= @all_prs_count > 1 ? "You have #{@all_prs_count} automatic dependency upgrade PRs open on the following apps:" : "You have 1 automatic dependency upgrade PR open on the following app:" %>
```